app/models/cloud_tenant.rb

```diff
@@ -8,7 +8,6 @@ class CloudTenant < ApplicationRecord
   has_many   :cloud_subnets
   has_many   :network_ports
   has_many   :network_routers
-  has_many   :shared_cloud_networks, -> { where :shared => true }, :through => :ext_management_system, :source => :cloud_networks
```

Comment:
it does not really work, tests revealed this, after I deleted the old relations from the base class. I can't really solve this by through since there is a belongs_to relation.